Sofa/GL/src/sofa/gl/DrawToolGL.cpp

```diff
@@ -144,19 +144,25 @@ void DrawToolGL::drawLines(const std::vector<Vector3> &points, float size, const
         return drawLines(points, size, RGBAColor::red());
     }
 
-    glLineWidth(size);
-    if (getLightEnabled()) disableLighting();
-    glBegin(GL_LINES);
+    // gather lines with same colors
+    std::map<RGBAColor, std::vector<Vector3> > colorPointsMap;
```

Comment:
an `unordered_map` should be faster

Reply:
At the beginning I used a unordered_map indeed, but I got some compile error (about compare/hashing) and I was lazy to solve it

Reply:
Indeed, a hash function should be available for `RGBAColor`. Keep using a map 👍